content/upbound-marketplace/providers.md

```diff
@@ -2,57 +2,100 @@
+Upbound builds, tests and supports official providers.
 
-{{<img src="upbound-marketplace/images/provider-by-upbound.png" alt="example official provider with a by Upbound seal" size="xtiny">}}
+Find official providers in the [Upbound Marketplace](https://marketplace.upbound.io/providers?tier=official). 
+All official providers have a purple `Official` tag.
 
-Also find official providers by filtering a [search in the marketplace](https://marketplace.upbound.io/providers?tier=official) to just `Official`.
+{{<img src="/upbound-marketplace/images/provider-by-upbound.png" alt="Official provider tag" >}}
 
-{{<img src="upbound-marketplace/images/official-provider-search-filter.png" alt="a marketplace search filter with Providers and Official filters set" size="xtiny" >}}
+## Provider families
+Upbound packages the official providers for Amazon AWS, Google GCP and Microsoft Azure as a "family." 
 
-## Install an official provider
-The [Upbound Marketplace](https://marketplace.upbound.io/) hosts official providers. 
+A provider family organizes the provider's resources into unique packages, organized by a common set of services.
 
-### Install the provider resource
-Install a provider with the creating a `Provider` Kubernetes resource with the [Up command-line]({{<ref "cli">}}) or manually.
+Provider families reduce the resources required to install and run providers and improve performance.
 
-{{< tabs "provider-install" >}}
+### Provider family requirements
 
-{{< tab "With the Up command-line" >}}
-Install the provider directly into Kubernetes using the Up command-line.  
-Provide the package source of the provider to install. This example installs the [AWS official provider version 0.17.0](https://marketplace.upbound.io/providers/upbound/provider-aws/v0.17.0).
-```shell {copy-lines="all"}
-up controlplane \
-provider install \
-xpkg.upbound.io/upbound/provider-aws:v0.17.0
-```
-{{< /tab >}}
+Provider families have the following requirements and restrictions:
+* [UXP]({{<ref "/uxp">}}) version 1.12.1-up or Crossplane version 1.12.1 or later
+* Remove existing [monolithic providers](#monolithic-official-providers) matching the new provider family. 
+
+{{<hint "warning" >}}
+If a provider family installs into a cluster with an existing monolithic provider, the family provider doesn't take effect. 
+
+The monolithic provider continues to own all provider resource endpoints until it's removed.
+
+For information on migrating from monolithic providers to provider families read the 
+[family providers migration guide]({{<ref "/knowledge-base/migrate-to-provider-families">}})
+{{< /hint >}}
 
-{{< tab "Using a Kubernetes manifest" >}}
-Create a `kind: Provider` Kubernetes manifest.  
+### Installing a provider family
 
-Provide the `spec.package` location of the official provider from the Upbound Marketplace listing. 
+Installing a provider family is identical to installing other Crossplane providers.
```

Comment:
Agreed that installation of family providers are identical to installation of any other providers. However, things get a bit complicated when there is a need to introduce a `ControllerConfig`, e.g. IRSA, since the base (or family, or config) provider is installed as a dependency if the user does not explicitly install it upfront. 

I believe we should document this as well which is quite a typical need to production scenarios.

Reply:
Do we have any notes on what's different that I can reference?

Reply:
I am not aware of any. Let me write down something:

When you install a service provider from a provider family (e.g. `provider-aws-s3`), the `provider-family` service provider (e.g. `provider-family-aws`) is installed as a dependency. However, there are certain cases where you would need to install the `provider-family` provider separately before installing other service providers.

- You use a `ControllerConfig` for some reason, e.g. IRSA or configuring resource requests/limits on the provider controller pod.

The procedure is roughly:

1. Create your `ControllerConfig`.
1. Install the `provider-family` service provider (e.g. `provider-family-aws`) with a reference to the `ControllerConfig`.
1. Wait until it is Installed & Healthy
1. Instal the remaining service providers with a reference to the `ControllerConfig`.

- You use a private registry where you've mirrored the providers you're interested in.

1. Install the `provider-family` service provider (e.g. `provider-family-aws`) pointing to your private registry.
1. Instal the remaining service providers with [skipDependencyResolution](https://doc.crds.dev/github.com/crossplane/crossplane/pkg.crossplane.io/Provider/v1@v1.12.1#spec-skipDependencyResolution).